ota-plus-web/conf/application.conf

```diff
@@ -92,7 +92,7 @@ websrc {
   style = "style-src 'unsafe-inline' 'self' "${websrc.bootstrapcdn}
   font = "font-src 'self' data: "${websrc.auth0}" "${websrc.bootstrapcdn}
   img = "img-src 'self' data: "${websrc.auth0}" "${websrc.gravatar}" "${websrc.githubusercontent}" "${websrc.googleusercontent}" "${websrc.wp}
-  connect = "connect-src 'self' "${websrc.atsgarage}" https://"${oauth.domain} ${?EXTRA_CONNECT_SRC}
+  connect = "connect-src 'self' "${websrc.atsgarage} ${?EXTRA_CONNECT_SRC}
```

Comment:
Why? we still have an app running with Auth0

Reply:
Ha right 😕 Changing the value of `oauth.domain` doesn't seem right though, because it's used in other places.
Should I better just keep this line as it is and set `EXTRA_CONNECT_SRC`?

Reply:
i think yes. We are going to get rid of auth0 soon, when it is done we will clean things up

Reply:
Reverted that and opened https://github.com/advancedtelematic/ota-charts/pull/250.